charts/jackett/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v0.16.2106
+appVersion: 0.17.591
 description: API Support for your favorite torrent trackers
 name: jackett
-version: 7.0.1
+version: 7.1.0
```

```suggestion
version: 7.0.2
```
I would be fine with only bumping the patch version for this

---

Agreed, this is not a feature bump